psalm.baseline.xml

```diff
@@ -16,12 +17,8 @@
     <LessSpecificReturnStatement occurrences="1">
       <code>$excludedNodes</code>
     </LessSpecificReturnStatement>
-    <MixedArgument occurrences="1">
-      <code>$m[0]</code>
-    </MixedArgument>
     <MixedArgumentTypeCoercion occurrences="2">
       <code>$attributeName</code>
-      <code>\usort($cssRules['inlinable'], [$this, 'sortBySelectorPrecedence'])</code>
```

Comment:
The `occurrences="2"` attribute has not changed but now there's only one occurrence.

Reply:
Ditto. I've re-generated the file.

Thinking of this: I'll try if deleting if before re-generating it helps.

Reply:
No, this doesn't change anything.

Reply:
Weird.  Looks like on the regeneration this one has been corrected, but different incorrect one has now arisen.